scene/gui/progress_bar.cpp

```diff
@@ -61,7 +61,7 @@ void ProgressBar::_notification(int p_what) {
 				Size2 size = get_size();
 				real_t fill_size = MIN(size.width, size.height) * 2;
 
-				if (Engine::get_singleton()->is_editor_hint() && !editor_preview_indeterminate) {
+				if (is_part_of_edited_scene() && !editor_preview_indeterminate) {
```

Comment:
These changes were needed to run indeterminate progress in Project Manager.
Looking at the code, I don't get how it already works in the editor (in template download).